bootstrap-salt.sh

```diff
@@ -4563,15 +4563,16 @@ install_freebsd_git_deps() {
     fi
     echodebug "Finished patching"
 
+    # Set _SALT_ETC_DIR to ports default
+    _SALT_ETC_DIR=${BS_SALT_ETC_DIR:-/usr/local/etc/salt}
+    # We also need to redefine the PKI directory
+    _PKI_DIR=${_SALT_ETC_DIR}/pki
+
     # Let's trigger config_salt()
     if [ "$_TEMP_CONFIG_DIR" = "null" ]; then
         _TEMP_CONFIG_DIR="${_SALT_GIT_CHECKOUT_DIR}/conf/"
-        CONFIG_SALT_FUNC="config_salt"
+        CONFIG_SALT_FUNC="config_freebsd_salt"
```

Comment:
That's a good point. I just changed it there since there is a specific config_freebsd_salt. I'll have to run a few more tests to make sure that will work like I think though. But, I'm assuming it should be fine.